packages/ui-kit/src/components/Logger/Log.types.ts

```diff
@@ -0,0 +1,23 @@
+export enum LogLevel {
+  Off = 'off',
+  Error = 'error',
+  Warn = 'warn',
+  Info = 'info',
+  Debug = 'debug'
+}
+
+export interface LogConfig {
```

Comment:
Nit: I do think Log or Logger would be a better name for this type. Can you change it? I think Log is better, since we can say

```ts
let log: Log
log = useLog()
```

the names match up this way. And we could also rename the Logger folder to Log. WDYT? Just using one name always is simpler: Log.